build_artifacts.sh

```diff
@@ -25,6 +25,15 @@ echo "Building to GOOS=linux GOARCH=arm GOARM=7"
 
 docker run --rm --env GOOS=linux --env GOARCH=arm --env GOARM=7 --env CGO_ENABLED=0 -v $(pwd):/code -w /code $GO_IMAGE go build -a -tags 'osusergo netgo static_build' -ldflags '-extldflags "-static"' -o dist/kool-linux-arm7
 
+echo "Building to GOOS=windows GOARCH=amd64"
+
+docker run --rm --env GOOS=windows --env GOARCH=amd64 --env CGO_ENABLED=0 -v $(pwd):/code -w /code $GO_IMAGE go build -a -tags 'osusergo netgo static_build' -ldflags '-extldflags "-static"' -o dist/kool.exe
+
+echo "Building kool-install.exe"
+
+docker run --rm -i -v $(pwd):/work amake/innosetup /dApplicationVersion=1.0.15 inno-setup/kool.iss
```

Comment:
@fabriciojs any suggestions for version?

Reply:
That is OK for now. We do have an open issue for dealing with hard-coded versions, we can deal with that later.